leakcanary-sample/build.gradle

```diff
@@ -50,3 +50,15 @@ android {
     }
   }
 }
+
+// Instrumentation test dependencies resolve to 27.1.1 so we align the sample.
+configurations.all {
+  resolutionStrategy {
+    eachDependency { details ->
+      // Force all of the primary support libraries to use the same version.
+      if (details.requested.group == 'com.android.support') {
+        details.useVersion "27.1.1"
+      }
+    }
+  }
+}
```

Comment:
why not put this in the root build.gradle instead with the error prone block?

Reply:
Wouldn't that change the support version of the entire project, effectively leading to the pom.xml being published with a dependency on 27, which I'm trying to avoid here?